examples/tutorials/advanced/websockets-example-MNIST/Federated learning with websockets and federated averaging.ipynb

```diff
@@ -20,7 +26,13 @@
     "\n",
     "Each worker is represented by two parts, a local handle (websocket client worker) and the remote instance that holds the data and performs the computations. The remote part is called a websocket server worker.\n",
     "\n",
-    "So first, we need to create the remote workers. For this, you need to run in a terminal (not possible from the notebook):\n",
+    "So first, you need to ```cd``` to the folder where this notebook and other additional files for running server and client are \n",
+    "\n",
+    "for example\n",
+    "\n",
+    "```cd PySyft-dev\\examples\\tutorials\\advanced\\websockets-example-MNIST```\n",
```

Comment:
Don't put windows path
or make it clear when this is for windows users (you can use citations with '>' . in markdown)

Example:
> **For Windows users**
> Use `cd my\path`

Reply:
done made it more generic